Makefile

```diff
@@ -69,6 +79,65 @@ build: ## Build manager binary
 docker-build: build ## Build the docker image
 	docker build . -t ${IMG}
 
+##@ Maintenance
+.PHONY: login-gh
+login-gh: ## Login to GitHub
+	if ! $(GH) auth status 2>/dev/null; then \
+		echo; \
+		echo '!! You need login to GitHub to proceed. Please follow the next command with "Authenticate Git with your GitHub credentials? (Y)".'; \
+		echo; \
+		$(GH) auth login -h github.com -p HTTPS -w; \
+	fi
+
+.PHONY: logout-gh
+logout-gh: ## Logout from GitHub
+	$(GH) auth logout
+
+.PHONY: update-contour
+update-contour: ## Update Contour and Kubernetes in go.mod
+	$(call get-latest-quay-tag,contour)
+	go get github.com/projectcontour/contour@$(call upstream-tag,$(latest_tag))
+	K8S_MINOR_VERSION="0."$$(go list -m -f '{{.Version}}' k8s.io/api | cut -d'.' -f2); \
+	K8S_PACKAGE_VERSION="$$(go list -m -versions k8s.io/api | tr ' ' '\n' | grep $${K8S_MINOR_VERSION} | sort -V | tail -n 1)"; \
+	go get k8s.io/api@$${K8S_PACKAGE_VERSION}; \
+	go get k8s.io/apimachinery@$${K8S_PACKAGE_VERSION}; \
+	go get k8s.io/client-go@$${K8S_PACKAGE_VERSION}; \
+	go mod tidy
+
+.PHONY: version
+version: login-gh ## Update dependent versions
+	$(call update-version,actions/checkout,ACTIONS_CHECKOUT_VERSION,1)
+	$(call update-version,actions/create-release,ACTIONS_CREATE_RELEASE_VERSION,1)
+	$(call update-version,actions/setup-go,ACTIONS_SETUP_GO_VERSION,1)
+	$(call update-version-quay,cert-manager,CERT_MANAGER_VERSION)
+	$(call update-version-quay,contour,CONTOUR_VERSION)
+	$(call update-version-quay,external-dns,EXTERNAL_DNS_VERSION)
+
+	$(call get-latest-quay-tag,argocd)
+	NEW_VERSION=$$(docker run quay.io/cybozu/argocd:$(latest_tag) kustomize version | cut -c2-); \
+	sed -i -e "s/KUSTOMIZE_VERSION := .*/KUSTOMIZE_VERSION := $${NEW_VERSION}/g" Makefile.versions
+
+	K8S_MINOR_VERSION="1."$$(go list -m -f '{{.Version}}' k8s.io/api | cut -d'.' -f2); \
+	NEW_VERSION=$$($(SETUP_ENVTEST) list | tr -s ' ' | cut -d' ' -f2 | fgrep $${K8S_MINOR_VERSION} | sort -V | tail -n 1 | cut -c2-); \
+	sed -i -e "s/ENVTEST_K8S_VERSION := .*/ENVTEST_K8S_VERSION := $${NEW_VERSION}/g" Makefile.versions
+
+.PHONY: update-actions
+update-actions:
+	$(call update-trusted-action,actions/checkout,$(ACTIONS_CHECKOUT_VERSION))
+	$(call update-trusted-action,actions/create-release,$(ACTIONS_CREATE_RELEASE_VERSION))
+	$(call update-trusted-action,actions/setup-go,$(ACTIONS_SETUP_GO_VERSION))
+
+.PHONY: maintenance
+maintenance: ## Update dependent manifests
+	$(MAKE) update-actions
+	$(MAKE) download-crds
+
+.PHONY: list-actions
```

Comment:
This target isn't used in other targets and doesn't appear in `maintenance.md`.
When are you supposed to use this?

Reply:
@terassyi 
This is a small usable function to check the completeness of the update script.
It is not used unless we're going to use a new Action.

If you suspect it is not needed, I'll move it to my `dotfiles`. How do you think?

Reply:
It is good for me to keep this in the file.
I just thought it might be a copy paste mistake. 🙇‍♂️